irispy/tests/test_iris_tools.py

```diff
@@ -1,2 +1,99 @@
 # -*- coding: utf-8 -*-
-# """Tests for functions in iris_tools.py"""
\ No newline at end of file
+# """Tests for functions in iris_tools.py"""
+from __future__ import print_function
+import pytest
+import irispy.iris_tools as iris_tools
+import numpy as np
+import numpy.testing as np_test
+
+source_data = np.array([[ 0.563,  1.132, -1.343],
+						[-0.719,  1.441, 1.566]])
+
+source_data1 = np.array([[1, 2, 3],
+						[4, 5, 6]])
+
+def test_convert_DN_to_photons_NUV():
+	"""
+	"""
+	# source_DN = [[ 0.563004,  1.132289, -1.343129], [-0.719726,  1.441411,  1.566724]]
```

Comment:
What's the commented out line?  Can it be deleted?  The same question for the other commented out ```source_xxx = []``` lines.